backends/tfhe-cuda-backend/cuda/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ endif()
 
 # Add OpenMP support
 find_package(OpenMP REQUIRED)
+find_package(mathdx REQUIRED COMPONENTS cufftdx CONFIG PATHS "/usr/include/nvidia/mathdx/22.11/")
```

Comment:
I'm wondering if this is the best solution, this seems platform specific?

Reply:
@IceTDrinker maybe you have an idea here?

Reply:
@bbarbakadze we should follow the indications from their documentation: https://docs.nvidia.com/cuda/cufftdx/installation.html. They recommend to have a copy of the header files in the `tfhe-cuda-backend` it seems.

Reply:
But we have to be careful with licensing issues. We should leave this PR as a draft for now until this question is addressed.

Reply:
we are already platform dependent through linux and "worse" even Ubuntu officially ? As long as they follow the standard linux filesystem organization (linux standard base or something) I don't think it's too bad potentially, the thing I'm less sure about is the hardcoded version

License questions need to be adressed right away.

I don't see how having the headers in our project is a good idea/practice ? 🤔 could be a way for them to "hijack" code bases

Reply:
@bbarbakadze we should use the 24.01 version which is the latest.

Reply:
I think I saw a thing in the Cmake to download the gtest thing, can it do the same for that lib ?

Reply:
Good idea, I'll look into it